CONTRIBUTING.md

```diff
@@ -8,35 +8,43 @@
 
 ## How to contribute
 
-opensds is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into opensds easier.
+opensds is Apache 2.0 licensed and accepts contributions via GitHub pull
+requests. This document outlines some of the conventions on commit message
+formatting, contact points for developers and other resources to make getting
+your contribution into opensds easier.
```

Comment:
@leonwanghui You said you have removed the doc changes, but there are still here?